clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1604,7 +1604,8 @@ raw2trace_t::append_bb_entries(raw2trace_thread_data_t *tdata,
         // TODO i#6102: This actually does the wrong thing for SIG_IGN interrupting
```

Comment:
So all this code should go away if we could figure out the root cause of the duplicate syscalls?  Which is #5934.  Please add #5934 to the Issue list in the PR description.